utils/vscode/carbon.code-snippets

```diff
@@ -0,0 +1,55 @@
+{
+  "package": {
+    "scope": "carbon",
+    "prefix": "package",
+    "body": "package $fileNameWithoutExt",
+    "description": "Carbon package"
+  },
+  "Main Run function": {
+    "scope": "carbon",
+    "prefix": "Run",
+    "body": ["fn Run() -> i32 {", "\t$1", "\treturn 0;", "}"]
+  },
+  "New Function": {
+    "scope": "carbon",
+    "prefix": "fn",
+    "body": ["fn $1($2) ->  $3 {", "\t$4", "}"]
+  },
+  "If statement": {
+    "scope": "carbon",
+    "prefix": "if",
+    "body": ["if ($1) {", "\t$2", "}"]
+  },
+  "Else if statement": {
+    "scope": "carbon",
+    "prefix": "elif",
+    "body": ["else if ($1) {", "\t$2", "}"]
+  },
+  "For each loop with for": {
+    "scope": "carbon",
+    "prefix": "foreach",
+    "body": ["for (var $1: $2 in $3) {", "\t$4", "}"]
+  },
+  "Match case": {
+    "scope": "carbon",
+    "prefix": "match",
+    "body": ["match ($1) {", "\tcase ($2) => {$3}", "\tdefault => {$4}"]
+  },
+  "Class declaration": {
+    "scope": "carbon",
+    "prefix": "class",
+    "body": [
+      "class $1 {",
+      "\tvar $2;",
+      "\tfn Make() -> Self {",
+      "\t\t$3",
+      "\t}",
+      "}"
```

Comment:
Class bodies don't seem like a good fit for snippets, because they don't have a fixed structure. In particular, a class body doesn't necessarily consist of a public data member followed by a `Make` function. In fact, that particular pattern seems likely to be fairly uncommon, because `Make` functions are most useful when the data members are private.

Reply:
Ok, I will probably remove it then